<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierEtudiant</class>
 <widget class="QDialog" name="ModifierEtudiant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier un étudiant</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>131</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelNom">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPrenom">
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelNumeroEtudiant">
      <property name="text">
       <string>N. étudiant</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDateDeNaissance">
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelNumeroTelephone">
      <property name="text">
       <string>N. téléphone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelAdresse">
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelFormation">
      <property name="text">
       <string>Formation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="nom"/>
    </item>
    <item>
     <widget class="QLineEdit" name="prenom"/>
    </item>
    <item>
     <widget class="QLineEdit" name="noEtudiant"/>
    </item>
    <item>
     <widget class="QLineEdit" name="dateDeNaissance"/>
    </item>
    <item>
     <widget class="QLineEdit" name="noTelephone"/>
    </item>
    <item>
     <widget class="QLineEdit" name="adresse"/>
    </item>
    <item>
     <widget class="QLineEdit" name="formation"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro de l'étudiant</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dernierNumeroEtudiant">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rechercher">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
